packages/next/src/build/swc/index.ts

```diff
@@ -25,6 +24,58 @@ const infoLog = (...args: any[]) => {
   Log.info(...args)
 }
 
+/**
+ * Based on napi-rs's target triples, returns triples that have corresponding next-swc binaries.
+ */
+const getSupportedArchTriples: () => Record<string, any> = () => {
+  const { darwin, win32, linux, freebsd, android } = platformArchTriples
+
+  return {
+    darwin,
+    win32,
+    linux: {
+      x64: linux.x64,
```

Comment:
Should this filter out the `gnux32` entry?

```js
{
  x64: [
    {
      platform: 'linux',
      arch: 'x64',
      abi: 'gnu',
      platformArchABI: 'linux-x64-gnu',
      raw: 'x86_64-unknown-linux-gnu'
    },
    {
      platform: 'linux',
      arch: 'x64',
      abi: 'gnux32',
      platformArchABI: 'linux-x64-gnux32',
      raw: 'x86_64-unknown-linux-gnux32'
    },
    {
      platform: 'linux',
      arch: 'x64',
      abi: 'musl',
      platformArchABI: 'linux-x64-musl',
      raw: 'x86_64-unknown-linux-musl'
    }
  ]
}
```

Reply:
argh you're right, somehow thought gnux32 is x32 arch. confusing enough.

Reply:
https://github.com/vercel/next.js/pull/53230/files#diff-907b7be0cfc75bd37773e5ebc38d1deffa40861b8ac1cde92e4992e284a1ee59R38